packages/cfd/src/features/Components/__tests__/personal-details-form.spec.js

```diff
@@ -0,0 +1,263 @@
+                    services: {
+                        idv: {
+                            documents_supported: {},
+                            has_visual_sample: 0,
+                            is_country_supported: 0,
+                        },
+                        onfido: {
+                            documents_supported: {},
+                            is_country_supported: 0,
+                        },
+                    },
+                },
+                phone_idd: '35818',
+                text: 'Aland Islands',
+                value: 'ax',
+            },
+        ],
+        initial_values: {
+            citizen: '',
+            tax_residence: '',
+            tax_identification_number: '',
+            account_opening_reason: '',
+            place_of_birth: '',
+        },
+    };
+
+    const citizenship_required_error = 'Citizenship is required';
+    const place_of_birth_required_error = 'Place of birth is required';
+    const tax_residence_required_error = 'Tax residence is required';
+    const tax_id_required_error = 'Tax identification number is required';
+    const opening_reason_required_error = 'Account opening reason is required';
+
+    it('should render properly on desktop', async () => {
+        render(<PersonalDetailsForm {...props} />);
+
+        expect(screen.getByTestId('dt_cfd_details_form_description')).toBeInTheDocument();
+        expect(screen.getByRole('textbox', { name: /citizenship/i })).toBeInTheDocument();
+        expect(screen.getByRole('textbox', { name: /tax residence/i })).toBeInTheDocument();
+        expect(screen.getByRole('textbox', { name: /tax identification number/i })).toBeInTheDocument();
+        expect(screen.getByText(/account opening reason/i)).toBeInTheDocument();
+        await waitFor(() => expect(screen.getByRole('button', { name: /next/i })).toBeDisabled());
+    });
+
+    it('should not render scrollbars or modal footer wrapper on mobile', () => {
+        isMobile.mockReturnValue(true);
+        render(<PersonalDetailsForm {...props} />);
+
+        expect(screen.queryByTestId('dt_themed_scrollbars')).not.toBeInTheDocument();
+        expect(screen.queryByTestId('dt_modal_footer')).not.toBeInTheDocument();
+        expect(screen.getByRole('button', { name: /next/i })).toBeInTheDocument();
+    });
+
+    it("should show that it's loading when is_loading is true", () => {
+        render(<PersonalDetailsForm {...props} is_loading />);
+        expect(screen.getByTestId('dt_initial_loader')).toBeInTheDocument();
+        expect(screen.queryByTestId('dt_cfd_details_form_description')).not.toBeInTheDocument();
+    });
+
+    it("should show that it's loading when residence_list is still empty", () => {
+        render(<PersonalDetailsForm {...props} residence_list={[]} />);
+
+        expect(screen.getByTestId('dt_initial_loader')).toBeInTheDocument();
+        expect(screen.queryByTestId('dt_cfd_details_form_description')).not.toBeInTheDocument();
```

Comment:
```suggestion
        expect(screen.queryByTestId('dt_details_form_description')).not.toBeInTheDocument();
```

Reply:
this will be handled int his card https://app.clickup.com/t/20696747/CFDS-1486